os/board/imxrt1050-evk/src/imxrt_boot.c

```diff
@@ -63,18 +63,80 @@
 #include <tinyara/pwm.h>
 #include <tinyara/gpio.h>
 #include <tinyara/spi/spi.h>
+#include <tinyara/analog/adc.h>
+#include <stdbool.h>
```

Comment:
Please locate <stdbool.h> and <sys/types.h> as shown below:
```
#include <tinyara/config.>

#include <sys/types.h>    <--- here
#include <stdbool.h>      <--- here
#include <stdint.h>
#include <stdio.h>
```

Reply:
relocated.